src/components/WebexAvatar/WebexAvatar.jsx

```diff
@@ -26,16 +48,28 @@ export default function WebexAvatar({
   const type = displayStatus ? status : null;
```

Comment:
status, type, icon name :)

I would suggest:
```js
const iconName = status;
```
this is easy now but might change in the future, I think a map will be needed soon from person statuses to icon names. At least it's obvious that we are using the status string as an icon name verbatim.